base/xx-info

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env sh
+
+: ${TARGETPLATFORM=}
+: ${TARGETOS=}
+: ${TARGETARCH=}
+: ${TARGETVARIANT=}
+
+: ${XX_MARCH=unknown}
+# https://pkgs.alpinelinux.org/packages
+: ${XX_ALPINE_ARCH=unknown}
+# https://www.debian.org/ports/
+: ${XX_DEBIAN_ARCH=unknown}
+: ${XX_TRIPLE=unknown-unknown-none}
+: ${XX_VENDOR=unknown}
+: ${XX_LIBC=}
+
+usage() {
+  cat >&2 <<EOT
+Usage: $(basename $0) [COMMAND]
+
+Commands:
+  alpine-arch    Print target architecture for Alpine package repositories
+  arch           Print target architecture for Docker
+  debian-arch    Print target architecture for Debian package repositories
+  env            Print XX_* variables defining target environment
+  is-cross       Exit cleanly if target is not native architecture
+  libc           Print used libc (musl or gnu)
+  march          Print target machine architecture, uname -m
```

Comment:
This is inconsistent with the other `*-arch` commands and not as obviously coming from `uname`. Would prefer `uname-arch`. And also wonder if we shouldn't reverse it to `arch-*` instead so it's `arch-alpine`, `arch-debian`, `arch-uname`, `arch-pkg`.